pkgs/development/libraries/sqlite-replication/default.nix

```diff
@@ -0,0 +1,31 @@
+{ sqlite, fetchFromGitHub, tcl }:
+
+let
+  version = "3.29.0+replication3";
+  name = "sqlite-${version}";
+
+in sqlite.overrideAttrs (oldAttrs: {
+  inherit name version;
+
+  src = fetchFromGitHub {
+    owner = "CanonicalLtd";
+    repo = "sqlite";
+    rev = "version-${version}";
+    sha256 = "1jfzpdxwa2lc584ay20gmyd284r4fllps2iwgbi3fdbzcz6g6knd";
+  };
+
+  nativeBuildInputs = [ tcl ];
+
+  configureFlags = oldAttrs.configureFlags ++ [
+      "--enable-replication"
+      "--disable-amalgamation"
+      "--disable-tcl"
+  ];
+
+  # TODO: Document how to get these values
```

Comment:
How to properly generate these values?

cc @dtzWill @jokogr

Reply:
These are the commit date and hash, so for `3.29.0+replication3` these would be:
`D 2019-08-19T10:56:12` and `e131dd7676fea1a0e101bfe0f25744bc9c4f9a69` respectively ([source](https://github.com/canonical/sqlite/commit/e131dd7676fea1a0e101bfe0f25744bc9c4f9a69)).

If I remember correctly, I couldn't build it or package it without setting so. Feel free to test if it works without setting those files.

Reply:
These are now in the sqlite-replication upstream repo so we don't need to track them in nixpkgs.